src/app/content/utils/archiveTreeUtils.ts

```diff
@@ -39,3 +39,36 @@ export const findArchiveTreeSection = (
     stripIdVersion(section.shortId) === stripIdVersion(pageId)
     || stripIdVersion(section.id) === stripIdVersion(pageId)
   );
+
+export const prevNextBookPage = (
+  book: {tree: ArchiveTree},
+  pageId: string
+): {} | undefined => {
+  const flattenTree = flattenArchiveTree(book.tree);
+  const currentSection = findArchiveTreeSection(book, pageId);
+
+  const sections: { [s: string]: object; } = {};
+
+  if ( flattenTree && currentSection ) {
+    const index = flattenTree.findIndex((currentSection) =>
+    stripIdVersion(currentSection.shortId) === stripIdVersion(pageId)
+    || stripIdVersion(currentSection.id) === stripIdVersion(pageId));
```

Comment:
this is the same comparison logic that is already in the `findArchiveTreeSection` above, we could separate it out by defining a new function like 

```js
const sectionMatcher = (targetPageId) => (currentSection) =>
    stripIdVersion(currentSection.shortId) === stripIdVersion(targetPageId)
    || stripIdVersion(currentSection.id) === stripIdVersion(targetPageId));
```

and using that in both places